datasets/BEGM11002_antwerpen/BEGM11002_antwerpen.derived.ad

```diff
@@ -134,7 +134,7 @@
 - electric_vehicle_profile_5_share = 0.0
 - solar_pv_profile_1_share = 1.0
 - solar_pv_profile_2_share = 0.0
-- annual_infrastructure_cost_gas = 4.988546861845945e-29
+- annual_infrastructure_cost_gas = 29099857.79
```

Comment:
Based on #2662 I thought that gas costs had only been decreased by 1e6. Instead I'm seeing that the values have been changed altogether in most datasets. Why is this the case?

Reply:
First, the values were based on the EU gas network costs scaled by number of residences (which were also based on the NL costs) and now they are directly based on the NL costs scaled by the total gas demand. So they have improved :)